src/infiniteblockmpo.jl

```diff
@@ -226,3 +225,83 @@ function local_mpo_blocks(
   end
   return vector
 end
+
+"""
+combineblocks_linkinds(Hcl::InfiniteBlockMPO, L, R)
+
+Fuse the non-site legs of the infiniteBlockMPO Hcl and the corresponding left L and right R environments.
+Preserve the corner structure.
+Essentially the inverse of splitblocks. It becomes useful for the very dense MPOs once get after compression sometimes.
+Hcl is modified on site, L and R are not.
+Input: Hcl the infinite MPO
+Optional  L   the left environment (a vector of  ITensors)
+R   the right environment (a vector of  ITensors)
+Output: Hcl, newL, newR the updated MPO and environments
+"""
+function combineblocks_linkinds(H::InfiniteBlockMPO; L=nothing, R=nothing)
```

Comment:
Were `L` and `R` supposed to be keyword arguments? I'm ok either way, just checking, since they are positional arguments in the `InfiniteMPO` version of `combineblocks_linkinds`. Also that version doesn't provide default values, seems like we should try to keep the interfaces consistent between the two.

Reply:
If we do decide they should be keyword arguments, we should choose better names for them, like `left_environment` and `right_environment`.